_data/sites.json

```
@@ -16373,6 +16373,19 @@
         ]
     },
 
+    {
+        "name": "PCS Mastercard",
+        "url": "https://www.mypcs.com/besoin-daide/supprimer-mon-compte/#faq-comment-supprimer-mon-compte-?",
+        "difficulty": "hard",
+        "email": "dpo@creacard.net",
+        "email_body": "I want to delete my PCS account.",
+        "notes": "Send an email to delete your account.",
+        "notes_tr": "Envoyez un email pour supprimer votre compte.",
```

Shouldn't this be french?
```suggestion
        "notes_fr": "Envoyez un email pour supprimer votre compte.",
```